Cargo.toml

```diff
@@ -85,6 +85,8 @@ tracing = "0.1"
 tracing-subscriber = "0.3"
 url = "2.4.1"
 zeroize = "1.6"
+trace4rs = "0.5.1"
```

Comment:
Do you know if this is available on Github somewhere? I can't find a link at https://crates.io/crates/trace4rs

Reply:
https://github.com/imperva/trace4rs